src/core/Microsoft.PowerToys.Settings.UI/ViewModels/KeyboardManagerViewModel.cs

```diff
@@ -113,17 +112,17 @@ public List<KeysDataModel> RemapKeys
             }
         }
 
-        public List<KeysDataModel> RemapShortcuts
+        public List<AppSpecificKeysDataModel> RemapShortcuts
         {
             get
             {
                 if (profile != null)
                 {
-                    return profile.RemapShortcuts.GlobalRemapShortcuts;
+                    return profile.RemapShortcuts.GlobalRemapShortcuts.ConvertAll(x => new AppSpecificKeysDataModel { OriginalKeys = x.OriginalKeys, NewRemapKeys = x.NewRemapKeys, TargetApp = "All Apps" }).Concat(profile.RemapShortcuts.AppSpecificRemapShortcuts).ToList();

```

Comment:
Since there was no tests for the KeyboardManager page in Settings, I've added a file for that and I moved this `ConvertAll` logic to a separate function so that I can test the method separately without having to create a `KeyboardManagerViewModel` object. Added 4 tests for the following scenarios - empty, empty; 1 entry, empty; empty, 1 entry, and 1 entry, 1 entry.